homework/src/main/java/io/codelex/kontroldarbs/exercise1/Card.java

```diff
@@ -0,0 +1,69 @@
+package io.codelex.kontroldarbs.exercise1;
+
+import java.math.BigDecimal;
+
+public abstract class Card {
+
+    private String cardNumber;
+    private String owner;
+    private String cvv;
+    private BigDecimal balance;
+
+    public Card(String cardNumber, String owner, String cvv, BigDecimal balance) {
+        if (!cvv.matches("\\d{3}")) {
```

Comment:
Validācija teorētiski nebija nepieciešama - apraksts uzdevumā ir tāds, lai varētu korekti izvēlēties datu tipu. :)

Reply:
It kā jā, bet tas apraksts bija tik precīzi definēts, ka gribējās validēt. Varu noņemt, ja nevajag, bet labprāt atstātu.